calico/maintenance/monitor/monitor-component-metrics.md

```diff
@@ -44,21 +44,40 @@ This tutorial will go through the necessary steps to implement basic monitoring
 > **Note**: This guide assumes {{site.prodname}} was installed in the kube-system namespace. If using the Operator, you should replace instances of kube-system with calico-system instead.
    {: .alert .alert-info}
 
-#### 1. Configure Calico to enable metrics reporting
+#### 1. Configure {{site.prodname}} to enable metrics reporting
 
 ##### **Felix configuration**
-Felix prometheus metrics are **disabled** by default. You have to manually change your Felix configuration (**prometheusMetricsEnabled**) via calicoctl in order to use this feature.
+Felix prometheus metrics are **disabled** by default. You have to manually change your Felix configuration (**prometheusMetricsEnabled**) via calicoctl or [Calico API server (Calico 3.20+)]({{ site.baseurl }}/maintenance/install-apiserver) in order to use this feature.
 
 > **Note**: A comprehensive list of configuration values can be [found at this link]({{ site.baseurl }}/reference/felix/configuration).
    {: .alert .alert-info}
 
+{% tabs %}
+<label:apiserver,active:true>
+<% 
+```bash
+kubectl patch felixConfiguration default --type merge --patch '{"spec":{"prometheusMetricsEnabled": true}}'
+```
+
+You should see an output like below:
+```
+felixconfiguration.crd.projectcalico.org/default patched
```

Comment:
nope, this output is bad and indicates a problem with the way you installed the apiserver, or your kubectl api version cache. 

It should be this:

```
felixconfiguration.projectcalico.org/default patched
```

Note the lack of "crd"

Reply:
Nice catch! got fixed after clearing kubectl cache.